ejb3/src/main/java/org/jboss/as/ejb3/inflow/MessageEndpointService.java

```diff
@@ -42,7 +42,7 @@
     /**
      * Returns the classloader that's applicable for the endpoint application.
      * This classloader will be used to set the thread context classloader during the beforeDelivery()/afterDelivery()
-     * callbacks as mandated by the JCA 1.6 spec
```

Comment:
I have changed `JCA 1.6` -> `Jakarta Connectors`
which is not a correct specification name according to https://jakarta.ee/specifications/connectors/
What should I put here?

Reply:
@maeste @tadamski ^

Reply:
@maeste @tadamski ^

Reply:
The point is the callbacks is mandatory starting from JCA 1.6. I think we should keep it as "JCA 1.6 and Jakarta Connectors >= 1.7"